docs/guide-build.md

```diff
@@ -52,6 +52,16 @@ go mod vendor
 ```
 Will enable use of vendor folder to share go dependencies. 
 
+## Installing the helm
+Before run examples make sure that you have [helm](https://github.com/helm/helm). To install helm you can use prepared scripts: 
+```bash
+sh scripts/install-helm.sh
+helm init
+sh scripts/helm-patch-tiller.sh
```

Comment:
why not use 'helm-init' target?
`.PHONY: helm-init
helm-init:
	helm init --wait && ./scripts/helm-patch-tiller.sh`